about.html

```diff
@@ -0,0 +1,103 @@
+              </li>
+            </ul>
+          </div>
+        </div>
+      </nav>
+    </div>
+    <div class="container-fluid d-flex flex-column" id="header-about">
+      <p class="pff-light col-12 col-md-9 s-color hello-about" id="hello-about">"Hello! We make you RESPONSIVE..."</p>
+      <h1 class="pff-bold col-12 col-md-9" id="h1-about">THE BEST WEB PAGE BUILDERS OF THE YEAR</h1>
+      <hr/>
+      <p class="sff-regular col-12 col-md-8 p-color" id="p-about">
+        Contrary to popular belief, Lorem Ipsum is not simply random text. 
+        It has roots in a piece of classical Latin literature from BC, 
+        making it over years old. Richard McClintock, a Latin professor 
+        at Hampden-Sydney College in Virginia.
+      </p>
+      <p class="sff-regular" id="opo-about">You can contact us in our Github repository for any question or suggestion</p>
+      <p class="pff-bold" id="deal-about"><a href="https://github.com/Dangelo-JAN">https://github.com/Dangelo-JAN</a></p>
+    </div>
+  </header>
+  <section class="container-fluid about-section">
+    <h3 class="h3-about pff-semibold p-color">Global Creative Designers logo</h3>
+    <hr/>
+    <p class="p-about-sec pff-semibold">
+      The Global Creative Designers logo was conceived through multiple
+      studies carried out in the field of responsive web design, with the 
+      need to show our clients the capacity that Global Creative Designers 
+      has in making a custom responsive and adaptable to all devices. and 
+      sizes of screens that exist in the technological market
+    </p>
+    <img class="logo-image" src="img/logo-gaf12d2114_1280.png" alt="logo"/>
+  </section>
+  <section class="container-fluid last-section">
+    <h3 class="h3-about pff-semibold p-color">Go through Global Creative Designers</h3>
+    <hr/>
+    <p class="p-about-sec pff-semibold">
+      take a moment and visit us at our offices so you can meet the 
+      great quality team we have
+    </p>
+    <div class="container-fluid center-images">
+      <img class="imagen-about" src="img/about_2011.png" alt="people"/>
+      <img class="imagen-about" src="img/about_2013.png" alt="people"/>
+    </div>
+  </section>
+  <section class="container-fluid d-flex flex-column bg-color margin-bottom">
+    <h2 class="t-color main-h2">Sponsors</h2>
+    <hr class="s-color"/>
+    <div class="d-flex justify-content-center">
+      <img class="sponsor-img" src="img/sponcer-1.png" alt="sponsor"/>
+      <img class="sponsor-img" src="img/sponcer-2.png" alt="sponsor"/>
+      <img class="sponsor-img" src="img/sponcer-3.png" alt="sponsor"/>
+    </div>
+    <div class="d-flex justify-content-center">
+      <img class="sponsor-img" src="img/sponcer-4.png" alt="sponsor"/>
+      <img class="sponsor-img" src="img/sponcer-5.png" alt="sponsor"/>
+    </div>
+  </section>
+  <footer class="container-fluid d-flex justify-content-center align-items-center" id="footer">
+    <img src="img/logo.png" alt="logoCC">
```

Comment:
- [ ] You haven't used the same logo for all the sections of your website, It would be nicer to use the same logo on all the sections of your website so your project looks more professional.

<img width="940" alt="image" src="https://user-images.githubusercontent.com/77190203/158039975-b7b6e4b2-a4a6-49b1-bc44-09c4b1259f56.png">


<img width="608" alt="image" src="https://user-images.githubusercontent.com/77190203/158039991-7506f62d-3629-45f6-b095-49872d8603e9.png">

<img width="944" alt="image" src="https://user-images.githubusercontent.com/77190203/158040014-fe2cf31f-96ec-4031-8a21-74553f44611b.png">

Reply:
For the style, this is the style that it like me, I accomplished all the requirements for the capstone.